utils.sh

```diff
@@ -28,6 +28,18 @@ function create_repo() {
     --repository-name ${repo} > /dev/null
 }
 
+function set_policy() {
+  local region=$1
+  RepositoryPermissionPolicy=$(printenv | grep RepositoryPermissionPolicy | sed 's/.*=//')
```

Comment:
Ahh this was probably from where I had originally tried stuffing this in the `credentials()` function instead. I'll switch it out.